.github/workflows/esmeta-test262.yml

```
@@ -0,0 +1,41 @@
+name: 'esmeta test262'
+
+on: [pull_request]
+
+jobs:
+  esmeta-test262:
+    name: 'esmeta test262'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v3
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 17
+      - name: Add to PATH
+        shell: bash
+        run: |
+          echo "ESMETA_HOME=$(pwd)/vendor/esmeta" >> $GITHUB_ENV
+      - name: download esmeta
+        run: |
+          mkdir -p "${ESMETA_HOME}"
+          git clone --branch v0.1.0-rc8 --depth 1 https://github.com/es-meta/esmeta.git "${ESMETA_HOME}"
+          cd "${ESMETA_HOME}" && git submodule update --init --depth 1
+      - name: build esmeta
```

```suggestion
      - name: build esmeta
         working-directory: ${{ env.ESMETA_HOME }}
```
i think this works and would avoid the need for the cd?